index.html

```diff
@@ -30,6 +31,12 @@
 					</ul>
 				</div>
 			</div>
+			<script>
+				function toggleDiv(id) {
+					var div = document.getElementById(id);
+					div.style.display = div.style.display == "none" ? "inline-block" : "none";
```

Comment:
Ein kleiner Exkurs:

Ein `<div>` hat standardmäßig `display: block;`. Ein `<span>` hat `display: inline-block`. So gibt es Block-Elemente und Inline-Elemente. Es kommt auch vor, dass `<div>`s mal als Inline verwendet werden.
Hier in dem Fall ist es initial von Haus aus ein Block-Element, das nach einem Hide zu einem Inline-Element sichtbar gemacht wird. Kann funktionieren, tut es ja auch, ist aber semantisch nicht optimal. Es könnte unter Umständen bei komplexeren Dingen zu ungewünschten Verhalten führen.